gen_pages.py

```diff
@@ -0,0 +1,133 @@
+import mkdocs_gen_files
+import requests
+import os
+import re
+
+# GitHub raw URL for application.properties
+ODOO_OPENMRS_GITHUB_URL = "https://raw.githubusercontent.com/ozone-his/eip-odoo-openmrs/7773cdb2783c2b7a5310e49ec79e979a65f2c1e0/odoo-openmrs/src/main/resources/config/application.properties"
+OPENMRS_ORTHANC_GITHUB_URL = "https://raw.githubusercontent.com/ozone-his/eip-openmrs-orthanc/45b2c6361da5afcb0bf281229f484e8bec7ae0f2/openmrs-orthanc/src/main/resources/config/application.properties"
+KEYCLOCK_SUPERSET_GITHUB_URL = ""
+OPENMRS_SENAITE_GITHUB_URL = "https://raw.githubusercontent.com/ozone-his/eip-openmrs-senaite/7184244ee52db4df85efab9fed9c63a2d993cbe1/senaite-openmrs/src/main/resources/config/application.properties"
+ERPNEXT_OPENMRS_GITHUB_URL = "https://raw.githubusercontent.com/ozone-his/eip-erpnext-openmrs/789020a15dbbb10f0bd5b2c994a1de2e82cf7ae7/erpnext-openmrs/src/main/resources/config/application.properties"
+
+mkdocs_config_points = {'mkdocs-config-name': 'Name', 'mkdocs-config-description': 'Description',
+                        'mkdocs-config-location': 'Location', 'mkdocs-config-possible-values': 'Possible Values',
+                        'mkdocs-config-default-value': 'Default Value'}
+
+mkdocs_block_end_identifier = '# /mkdocs-end'
+
+app_github_dict = {'Odoo-OpenMRS Flows': ODOO_OPENMRS_GITHUB_URL, 'OpenMRS-SENAITE Flows': OPENMRS_SENAITE_GITHUB_URL,
+                   'ERPNext-OpenMRS Flows': ERPNEXT_OPENMRS_GITHUB_URL,'OpenMRS-Orthanc Flows': OPENMRS_ORTHANC_GITHUB_URL}
```

Comment:
Does this kind of stuff not defeat the purpose a bit? I mean both the naming of each variable (eg. `ODOO_OPENMRS_GITHUB_URL`) and the writing of the string ahead of it (eg. `'Odoo-OpenMRS Flows'`).

I think we would like to end up in a situation where we provide a list of `eip-a-b` URLs and the script does all the magic.

Reply:
Yeah we can define this in the `application.properties` for each eip route but I think adding a new application to Ozone is not very frequent so we can have it hard coded here.